woocommerce/wc-admin
# Refactor Sidebar code to ActivityPanel

See #113.

This PR makes a few changes to the existing sidebar and layout code:

* Sidebar is now renamed to ActivityPanel. In addition, logic for per page hiding/visibility has been removed. The ActivityPanel behavior has changed per the mock-ups and should only open when a tab is clicked on.
* Both the Header component and ActivityCard component have been moved from `components/` to `layout/` to be near the ActivityPanel. All these components tie together and are more layout related than normal components.
* The Header and ActivityPanel are now sticky per the mockups and scroll with the page. Other than that, the design has not been implemented and things still look rough. More changes are coming in other PRs, including better mobile styles.

To Test:
* Verify that things are not totally broken on both woo-dash/analytics pages and existing WC pages. Note that there is no design work done here, as this PR is mostly about setting the stage for that work.
* Verify tab open/close behavior. i.e. clicking "Orders" opens "Orders". Clicking it again closes. Clicking "Orders" opens "Orders", but then clicking "Reviews" switches to "Reviews", etc. T
